cmake/ProjectVersionFromGitTag.cmake

```diff
@@ -51,6 +51,8 @@ function(get_version_from_git_tag _prefix)
                 set(VERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}.${VERSION_TWEAK}")
                 set(VERSION_FULL "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}-${VERSION_PRE_RELEASE}+p${VERSION_TWEAK}-g${VERSION_HASH}")
             else()
+                get_git_head_revision(VERSION_REF VERSION_HASH)
+                string(SUBSTRING "${VERSION_HASH}" 0 7  VERSION_HASH)            
```

Comment:
Hm. Actually, you have a trailing whitespace here...

Reply:
Removed trailing whitespace